job-dsls/jobs/FlowJob.groovy

```diff
@@ -395,7 +395,10 @@ DEPLOY_DIR=$WORKSPACE/deploy-dir
  -Dgwt.memory.settings="-Xmx4g -Xms1g -Xss1M"
 # (2) upload the content to remote staging repo
 mvn -B -e org.sonatype.plugins:nexus-staging-maven-plugin:1.6.5:deploy-staged-repository -DnexusUrl=https://repository.jboss.org/nexus -DserverId=jboss-releases-repository\\
- -DrepositoryDirectory=$DEPLOY_DIR -s $SETTINGS_XML_FILE -DstagingProfileId=15c3321d12936e -DstagingDescription="kie $kieVersion" -DstagingProgressTimeoutMinutes=40'''
+ -DrepositoryDirectory=$DEPLOY_DIR -s $SETTINGS_XML_FILE -DstagingProfileId=15c3321d12936e -DstagingDescription="kie $kieVersion" -DstagingProgressTimeoutMinutes=40
+# creates a file (list) of the last commit hash of each repository as handover for production
+touch hash.txt
+./droolsjbpm-build-bootstrap/script/git-all.sh log -1 --pretty=oneline >> hash.txt'''
```

Comment:
The `touch` is not needed. `>>` will create new file if none exists.

I would also change the file name to be more descriptive. Maybe `git-commit-ids.txt` or `git-commit-hashes.txt`?